sdk/internal/recording/recording.go

```diff
@@ -486,20 +482,22 @@ func (r RecordingOptions) HostScheme() string {
 }
 
 func getTestId(pathToRecordings string, t *testing.T) string {
+	if strings.HasSuffix(pathToRecordings, "/") {
```

Comment:
Go has a nice module for the path joining - not a huge deal but will save you some trouble in the future: https://pkg.go.dev/path@go1.17#Join